backtick issues

mairaw opened the pull request:
A few more double backtick issues

mairaw committed:
backtick issues

BillWagner commented on docs/csharp/programming-guide/concepts/linq/sorting-data.md:
```diff
@@ -50,7 +50,7 @@ foreach (string str in query)
   
 #### Primary Descending Sort  
- The next example demonstrates how to use the `orderby``descending` clause in a LINQ query to sort the strings by their first letter, in descending order.  
+ The next example demonstrates how to use the `orderby` `descending` clause in a LINQ query to sort the strings by their first letter, in descending order.  
```
nit: should `orderby descending` be under one code fence?

BillWagner commented on docs/csharp/programming-guide/concepts/linq/sorting-data.md:
```diff
@@ -98,7 +98,7 @@ foreach (string str in query)
   
 #### Secondary Descending Sort  
- The next example demonstrates how to use the `orderby``descending` clause in a LINQ query to perform a primary sort, in ascending order, and a secondary sort, in descending order. The strings are sorted primarily by length and secondarily by the first letter of the string.  
+ The next example demonstrates how to use the `orderby` `descending` clause in a LINQ query to perform a primary sort, in ascending order, and a secondary sort, in descending order. The strings are sorted primarily by length and secondarily by the first letter of the string.  
```
same comment.

rpetrusha commented on docs/csharp/misc/cs1954.md:
```diff
@@ -10,7 +10,7 @@ ms.assetid: bdec399e-c43d-46d3-a01b-ef3572786fe5
 The best overloaded method match 'method' for the collection initializer element cannot be used. Collection initializer 'Add' methods cannot have ref or out parameters.  
   
- For a collection class to be initialized by using a collection initializer, the class must have a `public``Add` method that is not a `ref` or `out` parameter.  
+ For a collection class to be initialized by using a collection initializer, the class must have a `public` `Add` method that is not a `ref` or `out` parameter.  
```
nit: rather than \`public\`, this could be just public (without backticks)

rpetrusha commented on docs/framework/configure-apps/file-schema/wcf/msmqintegration.md:
```diff
@@ -46,7 +46,7 @@ Specifies a MSMQ transport for custom binding.
   
 |Attribute|Description|  
 |---------------|-----------------|  
-|customDeadLetterQueue|A URI that indicates the location of the per-application dead letter queue, where messages that have expired or failed to be delivered to the application are transferred.<br /><br /> For messages that require ExactlyOnce assurances (that is, `exactlyOnce` is set to `true`), this attribute defaults to the system-wide transactional dead-letter queue in MSMQ.<br /><br /> For m ...
```
I wonder how the next several happened.

rpetrusha commented on docs/framework/data/adonet/ef/date-and-time-functions.md:
```diff
@@ -8,15 +8,15 @@ The .NET Framework Data Provider for SQL Server (SqlClient) provides date and ti
+|`DATEDIFF(datepart,startdate,enddate)`|Returns the number of date and time boundaries crossed between two specified dates.<br /><br /> **Arguments**<br /><br /> `datepart`: A `String` that represents the part of the date to calculate the difference.<br /><br /> `startdate`: A starting date for the calculation is an expression that returns a `DateTime`, or `DateTimeOffset`, or `Time` value with p ...
+|`DATENAME(datepart,date)`|Returns a character string representing the specified datepart of the specified date.<br /><br /> **Arguments**<br /><br /> `datepart`: A `String` that represents the part of the date on which to return a new value.<br /><br /> `date`: An expression that returns a `DateTime,` or `DateTimeOffset`, or `Time` value with precision = [0-7], or a character string in a date fo ...
+|`DATEPART(datepart,date)`|Returns an integer that represents the specified datepart of the specified date.<br /><br /> **Arguments**<br /><br /> `datepart`: A `String` that represents the part of the date on which to return a new value.<br /><br /> `date`: An expression that returns a `DateTime,` or `DateTimeOffset,` or `Time` value with precision = [0-7], or a character string in a date format. ...
+|`DAY(date)`|Returns the day of the specified date as an integer.<br /><br /> **Arguments**<br /><br /> `date`:An expression of type `DateTime` or `DateTimeOffset` with precision = 0-7.<br /><br /> **Return Value**<br /><br /> The day of the specified date as an `Int32`.<br /><br /> **Example**<br /><br /> `SqlServer.DAY(cast('6/9/2006' as DateTime))`|  
```
Good catch on Rreturns!

BillWagner reviewed (approved):
This LGTM @mairaw 

I had two small nits if you want to fix them before merging.

rpetrusha commented on docs/framework/data/adonet/ef/date-and-time-functions.md:
```diff
@@ -8,15 +8,15 @@ The .NET Framework Data Provider for SQL Server (SqlClient) provides date and ti
+|`DAY(date)`|Returns the day of the specified date as an integer.<br /><br /> **Arguments**<br /><br /> `date`:An expression of type `DateTime` or `DateTimeOffset` with precision = 0-7.<br /><br /> **Return Value**<br /><br /> The day of the specified date as an `Int32`.<br /><br /> **Example**<br /><br /> `SqlServer.DAY(cast('6/9/2006' as DateTime))`|  
 |`GETDATE()`|Produces the current date and time in SQL Server internal format for datetime values.<br /><br /> **Return Value**<br /><br /> The current system date and time as a `DateTime` with a precision of 3.<br /><br /> **Example**<br /><br /> `SqlServer.GETDATE()`|  
 |`GETUTCDATE()`|Produces the datetime value in UTC (Coordinated Universal Time or Greenwich Mean Time) format.<br /><br /> **Return Value**<br /><br /> The `DateTime` value with a precision of 3 in UTC format.<br /><br /> **Example**<br /><br /> `SqlServer.GETUTCDATE()`|  
 |`MONTH(` `date` `)`|Returns the month of the specified date as an integer.<br /><br /> **Arguments**<br /><br /> `date`:An expression of type `DateTime` or `DateTimeOffset` with precision = 0-7.<br /><br /> **Return Value**<br /><br /> The month of the specified date as an `Int32`.<br /><br /> **Example**<br /><br /> `SqlServer.MONTH(cast('6/9/2006' as DateTime))`|  
```
This one -- i.e., Month(date) -- should also be fixed.

rpetrusha commented on docs/framework/data/adonet/ef/language-reference/date-and-time-canonical-functions.md:
```diff
@@ -11,40 +11,40 @@ ms.assetid: 9628b74f-1585-436a-b385-8b02ed0cdd63
+|`DiffMonths(startExpression,endExpression)`|Returns the difference, in months, between `startExpression` and `endExpression`.<br /><br /> **Arguments**<br /><br /> `startExpression`, `endExpression`: `DateTime` or `DateTimeOffset`. **Note:**  `startExpression` and `endExpression` must be of the same type. <br /><br /> **Return Value**<br /><br /> An `Int32`.|  
+|`DiffYears(startExpression,endExpression)`|Returns the difference, in years, between `startExpression` and `endExpression`.<br /><br /> **Arguments**<br /><br /> `startExpression`, `endExpression`: `DateTime` or `DateTimeOffset`. **Note:**  `startExpression` and `endExpression` must be of the same type. <br /><br /> **Return Value**<br /><br /> An `Int32`.|  
+|`GetTotalOffsetMinutes(datetimeoffset)`|Returns the number of minutes that the `datetimeoffset` is offset from GMT. This is generally between +780 and -780 (+ or - 13 hrs). **Note:**  This function is supported in SQL Server 2008 only. <br /><br /> **Arguments**<br /><br /> A `DateTimeOffset`.<br /><br /> **Return Value**<br /><br /> An `Int32`.|  
+|`Hour (expression)`|Returns the hour portion of `expression` as an `Int32` between 0 and 23.<br /><br /> **Arguments**<br /><br /> A `DateTime, Time` and `DateTimeOffset`.<br /><br /> **Example**<br /><br /> `-- The following example returns 22.`<br /><br /> `Hour(cast('22:35:5' as DateTime))`|  
```
nit: Hour (expression) --> Hour(expression)

rpetrusha commented on docs/framework/data/adonet/ef/language-reference/math-canonical-functions.md:
```diff
@@ -10,15 +10,15 @@ ms.assetid: 6f6cddc6-b561-4ebe-84b6-841ef5b4113b
-|`Ceiling(` `value` `)`|Returns the smallest integer that is not less than `value`.<br /><br /> **Arguments**<br /><br /> A `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#EDM_CEILING](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs ...
+|`Abs(value)`|Returns the absolute value of `value`.<br /><br /> **Arguments**<br /><br /> An `Int16`, `Int32`, `Int64`, `Byte`, `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> `Abs(-2)`|  
+|`Ceiling(value)`|Returns the smallest integer that is not less than `value`.<br /><br /> **Arguments**<br /><br /> A `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#EDM_CEILING](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/entit ...
  [!code-sql[DP EntityServices Concepts#EDM_CEILING](../../../../../../samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts/tsql/entitysql.sql#edm_ceiling)]|  
```
unrelated to your PR, but important to fix: This is intended to be in the second column of the previous row (for Ceiling(value)).  (See https://review.docs.microsoft.com/en-us/dotnet/framework/data/adonet/ef/language-reference/math-canonical-functions?branch=pr-en-us-6494).

rpetrusha commented on docs/framework/data/adonet/ef/language-reference/math-canonical-functions.md:
```diff
@@ -10,15 +10,15 @@ ms.assetid: 6f6cddc6-b561-4ebe-84b6-841ef5b4113b
  [!code-sql[DP EntityServices Concepts#EDM_CEILING](../../../../../../samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts/tsql/entitysql.sql#edm_ceiling)]|  
-|`Floor(` `value` `)`|Returns the largest integer that is not greater than `value`.<br /><br /> **Arguments**<br /><br /> A `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#EDM_FLOOR](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/e ...
+|`Floor(value)`|Returns the largest integer that is not greater than `value`.<br /><br /> **Arguments**<br /><br /> A `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#EDM_FLOOR](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/entitys ...
  [!code-sql[DP EntityServices Concepts#EDM_FLOOR](../../../../../../samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts/tsql/entitysql.sql#edm_floor)]|  
```
Similarly, this belongs in the second column of the row above.

rpetrusha commented on docs/framework/data/adonet/ef/mathematical-functions.md:
```diff
@@ -8,31 +8,31 @@ The .NET Framework Data Provider for SQL Server (SqlClient) provides math functi
+|`ATAN(expression)`|Returns the arctangent value of the specified numeric expression.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.ATAN(9)`|  
+|`ATN2(expression, expression)`|Returns the angle, in radians, whose tangent is between the two specified numeric expressions.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.ATN2(9, 8)`|  
+|`CEILING(expression)`|Converts the specified expression to the smallest integer that is greater than or equal to it.<br /><br /> **Arguments**<br /><br /> `expression`: An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Return Value**<br /><br /> An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#SQLSERVER_CEILING](.. ...
  [!code-sql[DP EntityServices Concepts#SQLSERVER_CEILING](../../../../../samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts/tsql/entitysql.sql#sqlserver_ceiling)]|  
```
This belongs in the second column of the previous (Ceiling) row. See https://review.docs.microsoft.com/en-us/dotnet/framework/data/adonet/ef/mathematical-functions?branch=pr-en-us-6494.

rpetrusha commented on docs/framework/data/adonet/ef/mathematical-functions.md:
```diff
@@ -8,31 +8,31 @@ The .NET Framework Data Provider for SQL Server (SqlClient) provides math functi
+|`DEGREES(radians)`|Returns the corresponding angle in degrees.<br /><br /> **Arguments**<br /><br /> `expression`: An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Return Value**<br /><br /> An `Int32`, `Int64`, `Double`, or `Decimal`.<br /><br /> **Example**<br /><br /> `SqlServer.DEGREES(3.1)`|  
+|`EXP(expression)`|Calculates the exponential value of a specified numeric expression.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> `SqlServer.EXP(1)`|  
+|`FLOOR(expression)`|Converts the specified expression to the largest integer less than or equal to it.<br /><br /> **Arguments**<br /><br /> `expression`: A `Double`.<br /><br /> **Return Value**<br /><br /> A `Double`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#SQLSERVER_FLOOR](../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/c ...
  [!code-sql[DP EntityServices Concepts#SQLSERVER_FLOOR](../../../../../samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts/tsql/entitysql.sql#sqlserver_floor)]|  
```
This belongs in the second column of the previous (Floor) row.

rpetrusha reviewed (changes requested):
Looks very good, @mairaw. Left some nits, as well as one additional double backtick, and also (unrelated to your PR) some examples that are in the wrong table column

mairaw committed:
feedback

mairaw commented:
Thanks for your thorough review @BillWagner and @rpetrusha. I've addressed all your comments modulo the one I didn't understand from Ron. Please take a look and see if this is ready to merge!

rpetrusha reviewed (approved):
This is a vast improvement, @mariaw. You can merge when you're ready.
